packages/backend/src/models/table-column-metadata.ts

```diff
@@ -0,0 +1,35 @@
+import Base from './base'
+import TableMetadata from './table-metadata'
+
+class TableColumnMetadata extends Base {
+  id!: string
+  tableId!: string
+  name: string
```

Comment:
I thought we were doing array of strings? This way is also OK but it's not immediately clear how we derive column order. Are we relying on some ordering within the `data` attribute in dynamo?

e.g. `data` looks like this:
```
{
  columnData: [
    {columnId: 'abc-def', value: 'topkek'},
    {columnId: 'ghi-jkl', value: 'herp derp'},
  ]
}
```

and we will match `columnId` from dynamo with the `id` from `column_metadata` when rendering?

Reply:
i wont be dealing with re-ordering of columns yet but i think it'll just be having another column in tablecolumnmetadata table that stores the order. for now, i'll just sort by created at.